mocks/proxy.go

```diff
@@ -0,0 +1,164 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
I know Go makes it hard to avoid including autogenerated code, but... any way to avoid this without breaking `go test` out of the box?